core/src/cubos/debug_render.cpp

```diff
@@ -0,0 +1,183 @@
+#include <cubos/debug_render.hpp>
+
+#include <cubos/gl/render_device.hpp>
+
+#include <list>
+
+#include <glm/gtc/matrix_transform.hpp>
+#include <glm/gtx/transform.hpp>
+
+using namespace cubos;
+using namespace cubos::gl;
+
+static RenderDevice* debugRenderDevice;
+static ConstantBuffer mvpBuffer;
+static ShaderBindingPoint mvpBindingPoint, colorBindingPoint;
+static ShaderPipeline pipeline;
+static io::Window* debugWindow;
+
+static RasterState FillRasterState, WireframeRasterState;
+
+struct DebugDrawObject
+{
+    VertexArray va;
+    IndexBuffer ib;
+    int numIndices;
+};
+
+static DebugDrawObject objCube, objSphere;
+
+struct DebugDrawRequest
+{
+    DebugDrawObject obj;
```

Comment:
Shouldn't this be a pointer/reference? Right now you're copying the whole DebugDrawObject which is unnecessarily expensive.